src/modules/27-platform/default-settings/components/__test__/ReusableHandlers.test.tsx

```diff
@@ -102,7 +102,7 @@ describe('Reusable Components', () => {
     const { container } = renderObj
     expect(container).toMatchSnapshot()
   })
-  test('DefaultSettingTextbox', () => {
+  test('DefaultSettingMultiTextbox', () => {
```

Comment:
Looks like `DefaultSettingNumberTextbox` is used in the test
```suggestion
  test('DefaultSettingNumberTextbox', () => {
```

Reply:
yes @rameshx . Corrected test title.